.gitignore

```diff
@@ -4,3 +4,6 @@
 !programs-remote-tests/
 !circle.yml
 !readme.md
+!docker-compose.yml
```

Comment:
I don't like the reverse gitignore (ignoring everything except for certain files). It was useful for vargrant because it went into a directory that had other stuff in it, but now it's not necessary and can be made simpler. We'll need to do it for programs too.